funcs.py

```
@@ -0,0 +1,42 @@
+def doStuff(x):
+ for i in range(0, len(x)):
+  if x[i]%2==0:
+   print(str(x[i])+" is even")
+  else:print(str(x[i])+" is odd")
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)